holidays/countries/azerbaijan.py

```diff
@@ -136,23 +136,23 @@ def _add_observed(dt: date, name: str = None):
             # observed holidays special cases
             special_dates_obs = {2007: (JAN, 3), 2072: (JAN, 5)}
             if year in special_dates_obs:
-                hol_dates.add(
+                dts_observed.add(
```

Comment:
And here also `hol_dates`, otherwise it will be "Gurban Bayrami* (*estimated) (Observed) (Observed)" on 2007-01-05.